rocAL/rocAL/source/api/rocal_api_meta_data.cpp

```diff
@@ -77,13 +77,16 @@ ROCAL_API_CALL rocalCreateVideoLabelReader(RocalContext p_context, const char* s
 }
 
 RocalMetaData
-ROCAL_API_CALL rocalCreateCOCOReader(RocalContext p_context, const char* source_path, bool is_output, bool mask, bool ltrb, bool is_box_encoder) {
+ROCAL_API_CALL rocalCreateCOCOReader(RocalContext p_context, const char* source_path, bool is_output, bool _is_polygon_mask, bool is_pixelwise_mask, bool ltrb, bool is_box_encoder) {
     if (!p_context)
         THROW("Invalid rocal context passed to rocalCreateCOCOReader")
     auto context = static_cast<Context*>(p_context);
-    if(mask) {
+    if(_is_polygon_mask) {
```

Comment:
Check in DALI, Polygon masks and pixel wise masks are mutually exclusive so if both are set to true we need to throw error
Check DALI documentation and add error statement